core/src/main/java/org/mskcc/cbio/portal/scripts/ImportClinicalData.java

```diff
@@ -482,8 +487,21 @@ private int addPatientToDatabase(String patientId) throws Exception
 
     private int addSampleToDatabase(String sampleId, String[] fields, List<ClinicalAttribute> columnAttrs) throws Exception
     {
+        int sampleTypeIndex = findSampleTypeColumn(columnAttrs);
+        String sampleTypeStr = (sampleTypeIndex!=-1)?
+                fields[sampleTypeIndex]:null;
```

Comment:
Can this go on the same line, and maybe put spaces between the "?"operator and ":", just for style and consistency. And the "!=" operator.